src/EnergyPlus/AirflowNetwork/include/AirflowNetwork/Solver.hpp

```diff
@@ -171,181 +379,269 @@ namespace AirflowNetwork {
+        // error count during HVAC system operation | Exterior large opening error index during HVAC system operation | Large
+        // opening error count at Open factor > 1.0 | Large opening error error index at Open factor > 1.0 | Initialization flag
+        // type
+        Array1D<AirflowNetwork::AirflowNetworkNodeProp> AirflowNetworkNodeData;
+        Array1D<AirflowNetwork::AirflowNetworkCompProp> AirflowNetworkCompData;
+        Array1D<AirflowNetwork::AirflowNetworkLinkageProp> AirflowNetworkLinkageData;
+        Array1D<AirflowNetwork::MultizoneZoneProp> MultizoneZoneData;
+        Array1D<AirflowNetwork::MultizoneSurfaceProp> MultizoneSurfaceData;
+        Array1D<AirflowNetwork::DetailedOpening> MultizoneCompDetOpeningData;
+        Array1D<AirflowNetwork::SimpleOpening> MultizoneCompSimpleOpeningData;
+        Array1D<AirflowNetwork::HorizontalOpening> MultizoneCompHorOpeningData;
+        Array1D<AirflowNetwork::SurfaceCrack> MultizoneSurfaceCrackData;
+        Array1D<AirflowNetwork::EffectiveLeakageArea> MultizoneSurfaceELAData;
+        Array1D<AirflowNetwork::SpecifiedMassFlow> SpecifiedMassFlowData;
+        Array1D<AirflowNetwork::SpecifiedVolumeFlow> SpecifiedVolumeFlowData;
+        Array1D<AirflowNetwork::MultizoneExternalNodeProp> MultizoneExternalNodeData;
+        Array1D<AirflowNetwork::DeltaCpProp> DeltaCp;
+        Array1D<AirflowNetwork::DeltaCpProp> EPDeltaCP;
+        Array1D<AirflowNetwork::ZoneExhaustFan> MultizoneCompExhaustFanData;
+        Array1D<AirflowNetwork::IntraZoneNodeProp> IntraZoneNodeData;       // Intra zone data set
+        Array1D<AirflowNetwork::IntraZoneLinkageProp> IntraZoneLinkageData; // Intra zone linkage adat set
+        Array1D<AirflowNetwork::DisSysNodeProp> DisSysNodeData;
+        Array1D<AirflowNetwork::DuctLeak> DisSysCompLeakData;
+        Array1D<AirflowNetwork::EffectiveLeakageRatio> DisSysCompELRData;
+        Array1D<AirflowNetwork::Duct> DisSysCompDuctData;
+        Array1D<AirflowNetwork::Damper> DisSysCompDamperData;
+        Array1D<AirflowNetwork::ConstantVolumeFan> DisSysCompCVFData;
+        Array1D<AirflowNetwork::DetailedFan> DisSysCompDetFanData;
+        Array1D<AirflowNetwork::DisSysCompCoilProp> DisSysCompCoilData;
+        Array1D<AirflowNetwork::DisSysCompHXProp> DisSysCompHXData;
+        Array1D<AirflowNetwork::DisSysCompTermUnitProp> DisSysCompTermUnitData;
+        Array1D<AirflowNetwork::ConstantPressureDrop> DisSysCompCPDData;
+        Array1D<AirflowNetwork::AiflowNetworkReportProp> AirflowNetworkReportData;
+        Array1D<AirflowNetwork::PressureControllerProp> PressureControllerData;
+        Array1D<AirflowNetwork::OutdoorAirFan> DisSysCompOutdoorAirData;
+        Array1D<AirflowNetwork::ReliefFlow> DisSysCompReliefAirData;
+        Array1D<AirflowNetwork::AirflowNetworkLinkageViewFactorProp> AirflowNetworkLinkageViewFactorData;
+
+        void clear_state() override
+        {
+            OccupantVentilationControl.deallocate();
```

Comment:
I think the entire guts here could be replaced with `*this = Solver();`.  I'm a little surprised to see so many methods on this state class.  Not saying it's wrong, that's just not something we've done as yet.  We've been trying to thin out the state classes, but this could be part of the next evolution.

Reply:
I'm not sure I like the way that looks, but sure. Before the state refactor, I had broken out a solver object that was intended to do all the solving while the static data would reside elsewhere. This version contains all the data. Breaking some of that out again later would be fine but the separation of static and dynamic data is not fully complete. So I'd rather get it all gathered together so that more OOP patterns are possible and work on getting the static/dynamic thing sorted out on the fly.